post/templates/post/index.html

```diff
@@ -68,7 +68,13 @@
 
                                                 <!-- Read more: for long post -->
                                                 <a target="_blank" href="{% url 'post:detail' post.id %}">Read More</a>
-                                                <div><img src="#"></div>
+                                                <div>
+                                                    {% if post.image %}
+                                                        <img class="media-object" style="width: 48px; height: 48px;"
```

Comment:
Also, be aware that fixed width and height in pixels won't scale all too well with the overall responsive design. It is much better to use [Bootstrap grid](https://v4-alpha.getbootstrap.com/layout/grid/) over pixel perfection.